packages/mobile/src/screens/profile-screen/ProfileTabs/PlaylistsTab.tsx

```diff
@@ -50,7 +50,7 @@ export const PlaylistsTab = () => {
       disableTopTabScroll
       showsVerticalScrollIndicator={false}
       totalCount={playlist_count}
-      showCreatePlaylistTile={isOwner}
+      showCreateCollectionTile={isOwner}
```

Comment:
do we need reachability check here as well?

Reply:
yeah great call